### PR TITLE
Fix #130 with Cloudflare's SNI 

### LINKED ISSUE #130: Cloudflare remote error: tls: handshake failure
Hi. Get the issue with trying to load testing website behind the Cloudflare proxy. 
Config file:
```yml
pools:
  - id: HTTP pool                    # pool name (for your choice)
    gun:
      type: http
      target: target.site.com:443
      ssl: true
```
Error:
```
WARN	phttp/base.go:116	Request fail	{"pool": "HTTP pool", "instance": 7, "error": "remote error: tls: handshake failure"}
github.com/yandex/pandora/components/phttp.(*BaseGun).Shoot
	/home/icha1/Downloads/pandora/components/phttp/base.go:116
github.com/yandex/pandora/components/phttp.(*gunWrapper).Shoot
	/home/icha1/Downloads/pandora/components/phttp/core.go:43
github.com/yandex/pandora/core/engine.(*instance).shoot
	/home/icha1/Downloads/pandora/core/engine/instance.go:99
github.com/yandex/pandora/core/engine.(*instance).Run
	/home/icha1/Downloads/pandora/core/engine/instance.go:69
github.com/yandex/pandora/core/engine.runNewInstance
	/home/icha1/Downloads/pandora/core/engine/engine.go:407
github.com/yandex/pandora/core/engine.(*instancePool).startInstances.func2
	/home/icha1/Downloads/pandora/core/engine/engine.go:366
```
With another site (not behind cloudflare) everything work fine.

### PR DESCRIPTION
**Pull request by antonydevanchi**
fix issue #130 by PoC from #2a95367 and also few a little bit improvements

### CODE REVIEW

**Comment by antonydevanchi** [comment]
Affects Yandex.Cloud Load Testing

@ligreen @trueival

**Comment by antonydevanchi** [comment]
`$ pandora sni.yaml`

```yaml
# sni.yaml

pools:
  - id: HTTP
    gun:
      type: http
      target: example.tld:443
      ssl: true
    ammo:
      type: uri
      file: ../ammo.uri
    result:
      type: phout
      destination: ./phout.log
    startup:
      type: once
      times: 10
    rps:
      - duration: 60s
        type: line
        from: 1
        to: 10000
      - duration: 60s
        type: line
        from: 10000
        to: 100000
log:
  level: debug
```

```
# ../ammo.uri

[User-Agent: PSKNV]
[Connection: close]
[Accept: */*]
[Accept-Encoding: gzip, deflate]
https://example.tld/v1/health
https://example.tld/v2/health
```

Before:

```
2023-01-04T14:08:04.960Z	INFO	cli/cli.go:226	Reading config	{"file": "pandora_config_sd30pvwk.yaml"}
2023-01-04T14:08:04.982Z	DEBUG	cli/cli.go:271	Start monitoring	{"conf": {"Expvar":{"Enabled":true,"Port":1234},"CPUProfile":{"Enabled":false,"File":"cpuprofile.log"},"MemProfile":{"Enabled":false,"File":"memprofile.log"}}}
2023-01-04T14:08:04.982Z	INFO	engine/engine.go:139	Pool run started	{"pool": "HTTP"}
2023-01-04T14:08:04.983Z	DEBUG	engine/engine.go:304	Instances start awaited	{"pool": "HTTP", "started": 10}
2023-01-04T14:08:04.983Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 0}
2023-01-04T14:08:04.983Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 1}
2023-01-04T14:08:04.983Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 2}
2023-01-04T14:08:04.983Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 3}
2023-01-04T14:08:04.983Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 4}
2023-01-04T14:08:04.984Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 5}
2023-01-04T14:08:04.984Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 8}
2023-01-04T14:08:04.984Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 9}
2023-01-04T14:08:04.984Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 6}
2023-01-04T14:08:04.984Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 7}
2023-01-04T14:08:04.995Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 0, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:05.044Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 0, "URL": "https://188.114.99.224:443/v1/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:05.044Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 0, "Body": "<html>\r\n<head><title>403 Forbidden</title></head>\r\n<body>\r\n<center><h1>403 Forbidden</h1></center>\r\n<hr><center>cloudflare</center>\r\n</body>\r\n</html>\r\n"}
2023-01-04T14:08:05.044Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 0, "Status Code": 403, "Status": "403 Forbidden", "Headers": {"Cf-Ray":["78448b4f7ad32de5-DME"],"Content-Length":["151"],"Content-Type":["text/html"],"Date":["Wed, 04 Jan 2023 14:08:05 GMT"],"Server":["cloudflare"]}}
2023-01-04T14:08:05.099Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 4, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:05.145Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 1, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:05.146Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 4, "URL": "https://188.114.99.224:443/v1/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:05.146Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 4, "Body": "<html>\r\n<head><title>403 Forbidden</title></head>\r\n<body>\r\n<center><h1>403 Forbidden</h1></center>\r\n<hr><center>cloudflare</center>\r\n</body>\r\n</html>\r\n"}
2023-01-04T14:08:05.146Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 4, "Status Code": 403, "Status": "403 Forbidden", "Headers": {"Cf-Ray":["78448b502fc89d63-DME"],"Content-Length":["151"],"Content-Type":["text/html"],"Date":["Wed, 04 Jan 2023 14:08:05 GMT"],"Server":["cloudflare"]}}
2023-01-04T14:08:05.180Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 2, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:05.195Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 1, "URL": "https://188.114.99.224:443/v2/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:05.195Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 1, "Body": "<html>\r\n<head><title>403 Forbidden</title></head>\r\n<body>\r\n<center><h1>403 Forbidden</h1></center>\r\n<hr><center>cloudflare</center>\r\n</body>\r\n</html>\r\n"}
2023-01-04T14:08:05.195Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 1, "Status Code": 403, "Status": "403 Forbidden", "Headers": {"Cf-Ray":["78448b506d059d37-DME"],"Content-Length":["151"],"Content-Type":["text/html"],"Date":["Wed, 04 Jan 2023 14:08:05 GMT"],"Server":["cloudflare"]}}
2023-01-04T14:08:05.209Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 3, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:05.232Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 2, "URL": "https://188.114.99.224:443/v1/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:05.232Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 2, "Body": "<html>\r\n<head><title>403 Forbidden</title></head>\r\n<body>\r\n<center><h1>403 Forbidden</h1></center>\r\n<hr><center>cloudflare</center>\r\n</body>\r\n</html>\r\n"}
```

After:

```
2023-01-04T14:08:28.095Z	INFO	cli/cli.go:226	Reading config	{"file": "pandora_config_sd30pvwk.yaml"}
2023-01-04T14:08:28.112Z	DEBUG	cli/cli.go:271	Start monitoring	{"conf": {"Expvar":{"Enabled":true,"Port":1234},"CPUProfile":{"Enabled":false,"File":"cpuprofile.log"},"MemProfile":{"Enabled":false,"File":"memprofile.log"}}}
2023-01-04T14:08:28.112Z	INFO	engine/engine.go:139	Pool run started	{"pool": "HTTP"}
2023-01-04T14:08:28.114Z	DEBUG	engine/engine.go:304	Instances start awaited	{"pool": "HTTP", "started": 10}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 0}
2023-01-04T14:08:28.114Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 0, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 1}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 2}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 3}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 4}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 5}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 6}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 7}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 8}
2023-01-04T14:08:28.114Z	DEBUG	engine/instance.go:71	Instance started	{"pool": "HTTP", "instance": 9}
2023-01-04T14:08:28.218Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 1, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:28.233Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 0, "URL": "https://188.114.98.224:443/v1/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:28.233Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 0, "Body": "\u001f\ufffd\u0008\u0000\u0000\u0000\u0000\u0000\u0000\u0003\u00009\u0000\ufffd\ufffd{\"status\":\"Чувствую себя прекрасно\"}\u0000\u0000\u0000\ufffd\ufffd\u0003\u0000\ufffd\ufffd\ufffd\u001d9\u0000\u0000\u0000"}
2023-01-04T14:08:28.233Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 0, "Status Code": 200, "Status": "200 OK", "Headers": {"Alt-Svc":["h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"],"Cache-Control":["max-age=0, private, must-revalidate"],"Cf-Cache-Status":["DYNAMIC"],"Cf-Ray":["78448bdfffc616e3-DME"],"Content-Encoding":["gzip"],"Content-Type":["application/json; charset=utf-8"],"Date":["Wed, 04 Jan 2023 14:08:28 GMT"],"Etag":["W/\"3eb66bc852376e632ce8a53d0e37f7b8\""],"Nel":["{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}"],"Referrer-Policy":["strict-origin-when-cross-origin"],"Report-To":["{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=T47dxGq2esxCBS19Ky1eL9CpWCwlvmEdfydBZA6CpQasb2TcFwCdpcdvd1OIg1MviJDISjTy98BchCBeSHagJaZ5Gf67%2FtsH8y5DfQ1pCie24S5adpXtgv%2BMYZXVPjTg\"}],\"group\":\"cf-nel\",\"max_age\":604800}"],"Server":["cloudflare"],"Strict-Transport-Security":["max-age=15724800; includeSubDomains"],"Vary":["Accept, Origin"],"X-Content-Type-Options":["nosniff"],"X-Download-Options":["noopen"],"X-Frame-Options":["SAMEORIGIN"],"X-Permitted-Cross-Domain-Policies":["none"],"X-Request-Id":["5ab0a2d5a09b87bf852aa05d0821d8a5"],"X-Runtime":["0.033994"],"X-Xss-Protection":["1; mode=block"]}}
2023-01-04T14:08:28.264Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 2, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:28.298Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 3, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:28.327Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 4, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:28.331Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 1, "URL": "https://188.114.98.224:443/v2/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:28.331Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 1, "Body": "\u001f\ufffd\u0008\u0000\u0000\u0000\u0000\u0000\u0000\u0003\u00009\u0000\ufffd\ufffd{\"status\":\"Чувствую себя прекрасно\"}\u0000\u0000\u0000\ufffd\ufffd\u0003\u0000\ufffd\ufffd\ufffd\u001d9\u0000\u0000\u0000"}
2023-01-04T14:08:28.331Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 1, "Status Code": 200, "Status": "200 OK", "Headers": {"Alt-Svc":["h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"],"Cache-Control":["max-age=0, private, must-revalidate"],"Cf-Cache-Status":["DYNAMIC"],"Cf-Ray":["78448be0cb919d9f-DME"],"Content-Encoding":["gzip"],"Content-Type":["application/json; charset=utf-8"],"Date":["Wed, 04 Jan 2023 14:08:28 GMT"],"Etag":["W/\"3eb66bc852376e632ce8a53d0e37f7b8\""],"Nel":["{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}"],"Referrer-Policy":["strict-origin-when-cross-origin"],"Report-To":["{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=Udr3paIIIk7VaaJXYL6rStDtA2RupfsyIUeB3oEQNI9A55H%2FrqqB8Ige74nPioqADxZVX08zZeTdoZpq71c8k425%2BUkfdYM61%2FfgeE0FSlS8WpIaLfTumxhQZgKpjXyS\"}],\"group\":\"cf-nel\",\"max_age\":604800}"],"Retry-After":["3600"],"Server":["cloudflare"],"Strict-Transport-Security":["max-age=15724800; includeSubDomains"],"Vary":["Accept, Origin"],"X-Content-Type-Options":["nosniff"],"X-Download-Options":["noopen"],"X-Frame-Options":["SAMEORIGIN"],"X-Permitted-Cross-Domain-Policies":["none"],"X-Request-Id":["617757c5933fc2f1be851e3abe792a55"],"X-Runtime":["0.012790"],"X-Xss-Protection":["1; mode=block"]}}
2023-01-04T14:08:28.353Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 5, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:28.366Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 2, "URL": "https://188.114.98.224:443/v1/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:28.366Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 2, "Body": "\u001f\ufffd\u0008\u0000\u0000\u0000\u0000\u0000\u0000\u0003\u00009\u0000\ufffd\ufffd{\"status\":\"Чувствую себя прекрасно\"}\u0000\u0000\u0000\ufffd\ufffd\u0003\u0000\ufffd\ufffd\ufffd\u001d9\u0000\u0000\u0000"}
2023-01-04T14:08:28.367Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 2, "Status Code": 200, "Status": "200 OK", "Headers": {"Alt-Svc":["h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"],"Cache-Control":["max-age=0, private, must-revalidate"],"Cf-Cache-Status":["DYNAMIC"],"Cf-Ray":["78448be0fceb7b77-DME"],"Content-Encoding":["gzip"],"Content-Type":["application/json; charset=utf-8"],"Date":["Wed, 04 Jan 2023 14:08:28 GMT"],"Etag":["W/\"3eb66bc852376e632ce8a53d0e37f7b8\""],"Nel":["{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}"],"Referrer-Policy":["strict-origin-when-cross-origin"],"Report-To":["{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=a1jMIUeWPWn%2BTqEyLs3EfybS9gTMoPe99cCU36eqEbuSWb%2BVcXPrh5ARIv8mo%2F2kaoEp%2BSnfgIHZ%2BJsM5W9j9UnKumomMUsS%2B933r5r3cgvxLaf86OCqVm5WDgjDDk0%2B\"}],\"group\":\"cf-nel\",\"max_age\":604800}"],"Retry-After":["3600"],"Server":["cloudflare"],"Strict-Transport-Security":["max-age=15724800; includeSubDomains"],"Vary":["Accept, Origin"],"X-Content-Type-Options":["nosniff"],"X-Download-Options":["noopen"],"X-Frame-Options":["SAMEORIGIN"],"X-Permitted-Cross-Domain-Policies":["none"],"X-Request-Id":["ca61effed9db7dccf099bb210b6c5349"],"X-Runtime":["0.012060"],"X-Xss-Protection":["1; mode=block"]}}
2023-01-04T14:08:28.377Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 6, "url": "https://example.tld/v1/health"}
2023-01-04T14:08:28.398Z	DEBUG	phttp/base.go:129	Prepared ammo to shoot	{"pool": "HTTP", "instance": 7, "url": "https://example.tld/v2/health"}
2023-01-04T14:08:28.415Z	DEBUG	phttp/base.go:236	Request debug info	{"pool": "HTTP", "instance": 3, "URL": "https://188.114.98.224:443/v2/health", "Host": "example.tld", "Headers": {"Accept":["*/*"],"Accept-Encoding":["gzip, deflate"],"Connection":["close"],"User-Agent":["PSKNV"]}}
2023-01-04T14:08:28.415Z	DEBUG	phttp/base.go:248	Response body	{"pool": "HTTP", "instance": 3, "Body": "\u001f\ufffd\u0008\u0000\u0000\u0000\u0000\u0000\u0000\u0003\u00009\u0000\ufffd\ufffd{\"status\":\"Чувствую себя прекрасно\"}\u0000\u0000\u0000\ufffd\ufffd\u0003\u0000\ufffd\ufffd\ufffd\u001d9\u0000\u0000\u0000"}
2023-01-04T14:08:28.415Z	DEBUG	phttp/base.go:251	Response debug info	{"pool": "HTTP", "instance": 3, "Status Code": 200, "Status": "200 OK", "Headers": {"Alt-Svc":["h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400"],"Cache-Control":["max-age=0, private, must-revalidate"],"Cf-Cache-Status":["DYNAMIC"],"Cf-Ray":["78448be12cbe9d64-DME"],"Content-Encoding":["gzip"],"Content-Type":["application/json; charset=utf-8"],"Date":["Wed, 04 Jan 2023 14:08:28 GMT"],"Etag":["W/\"3eb66bc852376e632ce8a53d0e37f7b8\""],"Nel":["{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}"],"Referrer-Policy":["strict-origin-when-cross-origin"],"Report-To":["{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=9Y7er3DdqYQpHcYPxqfVFHc%2B3WXCC%2FSYR2vZ3gk2HFSEMy%2FOvSC6AwwYhUvKH4cx15NMHbzQO0F2S4zrQoSuZ5zKrlqGB3zRIoI0c5H%2F4XdNP88wYUCT11UlKb5326W%2B\"}],\"group\":\"cf-nel\",\"max_age\":604800}"],"Retry-After":["3600"],"Server":["cloudflare"],"Strict-Transport-Security":["max-age=15724800; includeSubDomains"],"Vary":["Accept, Origin"],"X-Content-Type-Options":["nosniff"],"X-Download-Options":["noopen"],"X-Frame-Options":["SAMEORIGIN"],"X-Permitted-Cross-Domain-Policies":["none"],"X-Request-Id":["63fe6f1f2cbc49a7da98cb6fcad6d269"],"X-Runtime":["0.010935"],"X-Xss-Protection":["1; mode=block"]}}
```

File: components/phttp/client.go [hunk 1]
```diff
@@ -112,13 +112,14 @@ func NewTransport(conf TransportConfig, dial netutil.DialerFunc) *http.Transport
 	tr.TLSClientConfig = &tls.Config{
 		InsecureSkipVerify: true,                 // We should not spend time for this stuff.
 		NextProtos:         []string{"http/1.1"}, // Disable HTTP/2. Use HTTP/2 transport explicitly, if needed.
+		ServerName:         strings.Split(target, ":")[0],
```
Comment (oke11o) [inline components/phttp/client.go:115]: We can use ipV6 target.
And this split will break such target

File: go.mod [hunk 2]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/atomic v1.3.1
 	go.uber.org/zap v1.7.1
-	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	golang.org/x/net v0.3.0
```
Comment (oke11o) [inline go.mod:24]: Changes to go.mod affect Yandex's internal automation. 
Could you undo the changes in these files for this PR

**Comment by antonydevanchi** [comment]
@oke11o fixed as requested

**Comment by antonydevanchi** [comment]
@oke11o I apologize for the intrusiveness, but can I ask about review status?

**Comment by oke11o** [comment]
@antonydevanchi 
I am very sorry. One more small request.
To save your authorship, you need to create PR to arcadia branch. Now PR to develop branch.
In the future, we will write a good doc about the pandora release cycle.

**Comment by antonydevanchi** [comment]
@oke11o https://github.com/yandex/pandora/pull/158

**Comment by antonydevanchi** [comment]
> In the future, we will write a good doc about the pandora release cycle.

No problem.
Anyway it is still much more friendly and transparently interaction then mostly corp's OSS repos :D

**Pull request closed without merging**